doc/conf.py

```diff
@@ -537,7 +538,7 @@ def add_js_css_files(app, pagename, templatename, context, doctree):
 # [howto/manual]).
 latex_documents = [
     (
-        "contents",
+        root_doc,
```

Comment:
This seems clearer
```suggestion
        "index",
```

Reply:
Thanks for the context, it seems reasonable to change tex engine, though we should probably update mentions of pdftex.

This seems like the minimal changes to allow the pdf to at least build.

>  manual action is required to fix these six issues and correctly build PDF.

yes I think this is why we decided to stop building pdf.